resources/api/openapi.yml

```diff
@@ -187,6 +221,91 @@ components:
                     type: string
                     example: 'https://example.com/summary'
 
+        ConfirmPreOrderPaymentRequest:
+            type: object
+            required:
+                - orderData
+                - orderReference
+                - paymentServiceProviderData
```

Comment:
This data can be different from PSP to PSP. It will be provided on the App side and projects can use it. I think we should have the default implementation to use this data developed in SCOS anyway. 

I will check if I can make the transactionId as default required param, I see it could make sense.